server/lib/dispatch.js

```diff
@@ -1,9 +1,10 @@
 import Irys from '@irys/sdk'
+import { createDataItemSigner, message } from '@permaweb/aoconnect'
 import fs from 'fs'
 
 const key = JSON.parse(fs.readFileSync(process.env.WALLET, 'utf-8'))
 
-export async function dispatch({ address }) {
+export async function dispatchToIrys({ address }) {
```

Comment:
This should be called either dispatch or dispatch to Arweave, it is an implementation detail that it uses Irys. preferred is `dispatch`

Reply:
Reverted